packages/frontend/src/hooks/useBigNumberConfig.ts

```diff
@@ -54,39 +55,42 @@ const formatComparisonValue = (
     }
     switch (format) {
         case ComparisonFormatTypes.PERCENTAGE:
-            return `${prefix}${formatValue(value, {
-                format: Format.PERCENT,
+            return `${prefix}${applyCustomFormat(value, {
                 round: 0,
+                type: CustomFormatType.PERCENT,
```

Comment:
is there a way we can pass the prefix to the `applyCustomFormat`?

Reply:
We only support adding `prefix` via format to `CustomFormatType.NUMBER` types. This one is taking in a `CustomFormatType.PERCENT` so it is not possible.